xcm/xcm-executor/src/lib.rs

```diff
@@ -42,23 +42,23 @@ pub use config::Config;
 
 /// The XCM executor.
 pub struct XcmExecutor<Config: config::Config> {
-	holding: Assets,
-	origin: Option<MultiLocation>,
-	original_origin: MultiLocation,
-	trader: Config::Trader,
+	pub holding: Assets,
+	pub origin: Option<MultiLocation>,
+	pub original_origin: MultiLocation,
+	pub trader: Config::Trader,
```

Comment:
Note that I could also create `runtime-benchmark` only getters and setters, but i dont see the harm